libcontainer/process_linux.go

```diff
@@ -290,6 +290,16 @@ func (p *initProcess) start() error {
 		p.process.ops = nil
 		return newSystemErrorWithCause(err, "starting init process command")
 	}
+	defer func() {
+		if retErr != nil {
+			// TODO: should not be the responsibility to call here
```

Comment:
Is this TODO still correct?

Reply:
@AkihiroSuda 
I have dropped this TODO.